frontend/components/index.ts

```diff
@@ -60,6 +60,4 @@ import './utils/js/utils.filter';
 import './viewer/js/multi-select.service';
 import './viewer/js/viewer.component';
 import './viewer/js/viewer.service';
-import './viewpoints/js/viewpoints.component';
```

Comment:
Should delete the viewpoints angular component folder also.

Reply:
done